doc/src/site/sphinx/configuration/configuration_properties.rst

```diff
@@ -146,19 +146,19 @@ Configuration properties independent of selected backend
 |                                                    |                | corresponding parameters in Sparkling  |
 |                                                    |                | Water.                                 |
 +----------------------------------------------------+----------------+----------------------------------------+
-| **H2O client parameters**                          |                |                                        |
-+----------------------------------------------------+----------------+----------------------------------------+
-| ``spark.ext.h2o.client.flow.dir``                  | ``None``       | Directory where flows from H2O Flow    |
-|                                                    |                | are saved.                             |
-+----------------------------------------------------+----------------+----------------------------------------+
-| ``spark.ext.h2o.client.flow.extra.http.headers``   | ``None``       | Extra HTTP headers that will be used   |
```

Comment:
@jakubhava Any idea how to properly deprecate `spark.ext.h2o.client.flow.extra.http.headers`?

Reply:
We can use https://github.com/h2oai/sparkling-water/blob/6fa71b8053630afd931b1abbba018cec70e92275/core/src/main/scala/org/apache/spark/h2o/H2OConf.scala#L116, what do think?

Reply:
That's smart, thanks! Will deprecate it in another PR.